Airbnb.Infrastructure/Services/ApartmentService.cs

```diff
@@ -0,0 +1,27 @@
+using Airbnb.Application.DTOs;
+using Airbnb.Application.Interfaces;
+using Airbnb.Data;
+using Airbnb.Domain.Entities;
+using Mapster;
+
+namespace Airbnb.Infrastructure.Services;
+
+public class ApartmentService : IApartmentService
+{
+    private readonly AppDbContext _context;
+
+    public ApartmentService(AppDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<ApartmentDto> CreateApartmentAsync(ApartmentDto dto)
```

Comment:
We should use different DTOs for Create/Get/Update methods, bc they are independent. Now your code works OK, but eventually your models will become different and you'll need refactoring. So do it now